pkg/f6106ec8-b651-4c2c-a92b-acce4fac2d1b/package.lp

```diff
@@ -0,0 +1,76 @@
+(librepcb_package f6106ec8-b651-4c2c-a92b-acce4fac2d1b
+ (name "Panasonic ALQ1xx")
```

Comment:
I know that the other two packages don't follow that convention yet, but according to IPC the package name should be `PANASONIC_ALQ1xx`.